components/inputs/docs/input-textarea.md

```diff
@@ -23,12 +23,14 @@ The `<d2l-input-textarea>` is a wrapper around the native `<textarea>` element t
 | `disabled` | Boolean | Disables the `textarea` |
 | `label` | String, required | Label for the `textarea` |
 | `label-hidden` | Boolean | Hides the label visually (moves it to the `textarea`'s `aria-label` attribute) |
-| `max-lines` | Number, default: 11 | Maximum number of lines before scrolling. Less than 1 allows `textarea` to grow infinitely. |
-| `min-lines` | Number, default: 5 | Minimum number of lines. If `min-lines` and `max-lines` are equal then auto-grow will be disabled. |
+| `max-rows` | Number, default: 11 | Maximum number of lines before scrolling. Less than 1 allows `textarea` to grow infinitely. |
```

Comment:
nit: should the description be changed to refer to rows instead of lines?

Reply:
sure yep... that would avoid some confusion.